script/test/Swap.sol

```diff
@@ -29,6 +42,7 @@ contract SwapTest is Script {
     // Get proxy addresses of the deployed tokens
     cUSD = contracts.celoRegistry("StableToken");
     cEUR = contracts.celoRegistry("StableTokenEUR");
+    usdCet = contracts.dependency("MockUSDCet");
```

Comment:
Is there a reason we're pulling this from the dependency instead of the [deployed contracts](https://github.com/mento-protocol/mento-deployment/blob/971caff93f794079539bdb84c5134fb79c07e9eb/script/utils/Contracts.sol#L74)?

Reply:
replied in your other comment